sematic/api/endpoints/runs.py

```diff
@@ -191,6 +196,21 @@ def _make_cursor(key: str) -> str:
     return base64.urlsafe_b64encode(bytes(key, "utf-8")).decode("utf-8")
 
 
+def _generateSearchPredicate() -> Optional[BooleanClauseList]:
```

Comment:
Python uses snake case.
Also, please only use `flask.request` inside the endpoint itself. The function should accept a string and return predicates. Extracting the string from the request is the concern of the endpoint.
```suggestion
def _generate_search_predicate(search_string: str) -> Optional[BooleanClauseList]:
```